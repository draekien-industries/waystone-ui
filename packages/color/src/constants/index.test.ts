import { COLOR_STOPS, THEME_COLOR_STOPS } from '.';

describe('Color constants', () => {
  it('theme color stops should match snapshot', () => {
    expect(THEME_COLOR_STOPS).toMatchSnapshot();
  });

  it('color stops should match snapshot', () => {
    expect(COLOR_STOPS).toMatchSnapshot();
  });
});
