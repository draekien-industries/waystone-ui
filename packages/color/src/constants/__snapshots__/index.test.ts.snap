// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Color constants color stops should match snapshot 1`] = `
[
  0,
  50,
  100,
  200,
  300,
  400,
  500,
  600,
  700,
  800,
  900,
  950,
  1000,
]
`;

exports[`Color constants theme color stops should match snapshot 1`] = `
[
  50,
  100,
  200,
  300,
  400,
  500,
  600,
  700,
  800,
  900,
  950,
]
`;
